src/components/Character/styles.scss

```diff
@@ -1,19 +1,189 @@
+// Mobile portrait
 .wrapper {
-	font-size: 1em;
+	display: flex;
+	position: absolute;
+	width: 100%;
+	min-height: 100%;
+	top: 0;
+	left: 0;
+	padding: 10%; 		//GUESS
+	flex-direction: column;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
+		pointer-events: none
 	}
 }
-	.button {
-		&.next {
-			font-size: 1em;
+
+	.container {
+		position: relative;
+		z-index: 1;
+	}
+
+		.content {
+			position: relative;
+			margin-top: 40vh;
+		}
+	
+			.name {
+				margin-bottom: 0.1em; // GUESS
+				font-size: 5em;
+				font-family: var(--font_family__heading);
+			}
+
+			.nickname {
+				margin-bottom: 0.5em; // GUESS
+				font-size: 2.6em;
+				font-family: var(--font_family__heading);
+			}
+
+			.actor {
+				margin-bottom: 1em; // GUESS
+				font-size: 1em;
+				font-family: var(--font_family__body);
+				font-weight: normal;
+				letter-spacing: 0.25em;
+				text-transform: uppercase;
+			}
+
+			.body {
+				margin-bottom: 4em; // GUESS
+				line-height: 1.5;
+				font-family: var(--font_family__body);
+			}
+
+		.content::after {
+			content: "";
+			display: block;
+			position: fixed;
+			width: 100%;
+			height: 12em;
+			bottom: 0;
+			left: 0;
+			background-image: linear-gradient(
+			    rgba(237, 20, 61, 0) 0%,
+			    rgba(237, 20, 61, 1) 50%,
+			    rgba(237, 20, 61, 1) 100%	
+			);
+		}
+
+		.controls {
+			display: flex;
+			position: sticky;
+			width: 100%;
+			max-width: 25rem; // GUESS
+			bottom: 10vw; // GUESS!
```

Comment:
maaate, what's going on with these CSS declaration orders? you've been nailing it up to now! 😂

Reply:
Huh? 🤔 We've been doing this no?
Display
Position
Width/Height
Top/Right/Bottom/Left

Reply:
I did what you suggested in the other one, and I feel dirty 🤧

Reply:
naww, remember, it's :
- inherited (flex-grow, shrink, basis, grid-columns etc) 
- box model (display, height, width, paddings, margins) 
- position (relative, fixed etc ; top, right, bottom, left, z-index) 
- layout (flex direction, align items, justify, text-align) 
- foreground (color, font weight, text transform, text shadow etc) 
- background (background color, image, size, box-shadow, repeat etc) 
- cursor (pointer events, cursor icon etc) 
transforms (translate rotate etc etc) 
- filters (blur, drop-shadow, brightness, saturation) 
- transitions 
- animations

Reply:
There are many places I haven't been doing that order, and not just in this project loool 🙊

Reply:
I'll keep an eye out going forward!